crates/sui-types/src/execution.rs

```diff
@@ -170,6 +171,12 @@ impl ExecutionResultsV2 {
     }
 }
 
+pub enum ExecutionTiming {
+    Success(Duration),
+    Abort(Duration),
+}
+pub type TimingResult<R, E> = Result<(R, Vec<ExecutionTiming>), (E, Vec<ExecutionTiming>)>;
```

Comment:
nit - this name makes it sounds like the main output of the result is the timing, maybe `ResultWithTiming`?

Reply:
done